util/appveyor/requirements.txt

```diff
@@ -0,0 +1,14 @@
+# Fetch a numpy wheel from the sklearn rackspace wheelhouse.
+# That wheel was generated by @ogrisel by calling `wheel convert` on
+# the binaries from http://www.lfd.uci.edu/~gohlke/pythonlibs/
+# This is a temporary solution. As soon as numpy provides an official
+# wheel for windows we ca delete this --find-links line.
+--find-links http://28daf2247a33ed269873-7b1aad3fab3cc330e1fd9d109892382a.r6.cf2.rackcdn.com/index.html
```

Comment:
I don't see a Cython wheel at the provided link (just some numpy and scipy ones):
http://28daf2247a33ed269873-7b1aad3fab3cc330e1fd9d109892382a.r6.cf2.rackcdn.com/index.html
perhaps we need a separate link to a location providing that?  I don't have any prior experience for wheels or appveyor so I am not sure.